catalyst/contrib/dl/callbacks/periodic_loader_callback.py

```diff
@@ -55,16 +58,20 @@ def __init__(self, **kwargs):
 
         self.valid_loader: str = None
         self.valid_metrics: Mapping[str, float] = None
+        self.best_valid_metrics: Mapping[str, float] = None
```

Comment:
why do we need so many `valid_metrics`?
from my point of view, we only need to store last `valid_metrics`, isn't it?